assets/multiqc_config.yml

```diff
@@ -315,6 +347,8 @@ extra_fn_clean_exts:
   - ".bbduk"
   - ".unmapped"
   - "_filtered"
+  - "porechop.log"
+  - "porechop_abi.log"
```

Comment:
```suggestion
  - "porechop"
  - "porechop_abi"
```

Reply:
Untested!

Reply:
Yah!!! I think sample names are correct now. 
Please double check.
[multiqc_report_porechop_abi.html.zip](https://github.com/user-attachments/files/16508473/multiqc_report_porechop_abi.html.zip)
[multiqc_report_porechop.html.zip](https://github.com/user-attachments/files/16508475/multiqc_report_porechop.html.zip)

Reply:
yaaay! Good to go now!